src/Sulu/Bundle/AdminBundle/Resources/js/services/Router/Router.js

```diff
@@ -8,23 +9,26 @@ export default class Router {
     history: Object;
     @observable currentRoute: Route;
     @observable currentParameters: Object;
+    @observable currentSearchParameters: Object;
```

Comment:
what's the difference between `parameters` and `searchParameters`? Are `searchParameters` the query Parameters? maybe rename to `queryParameters`?